dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/TaskInstance.java

```diff
@@ -84,6 +82,12 @@
      */
     private ExecutionStatus state;
 
+    /**
+     * task first submit time
+     */
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss",timezone="GMT+8")
```

Comment:
```suggestion
    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone="GMT+8")
```
Please configure `checkstyle.xml` and `code style xml` in style directory. After configure that, you can use `option+command+L` to automatic format the code.

Checkstyle.xml configuration lesson: https://www.cnblogs.com/wanshi1989/p/5478050.html
code style xml configuration lesson: https://blog.csdn.net/qq_34579060/article/details/80100040